task/03-date-tasks.js

```diff
@@ -76,14 +80,26 @@ function isLeapYear(date) {
  *    Date(2000,1,1,10,0,0),  Date(2000,1,1,15,20,10,453)   => "05:20:10.453"
  */
 function timeSpanToString(startDate, endDate) {
-   throw new Error('Not implemented');
+  let dif = new Date(endDate - startDate);
+  let hours =
+    startDate.getUTCDate() === endDate.getUTCDate()
+      ? dif.getUTCHours()
+      : dif.getUTCHours() + 24;
+  return `${hours < 10 ? "0" + hours : hours}:${
+    dif.getUTCMinutes() < 10 ? "0" + dif.getUTCMinutes() : dif.getUTCMinutes()
+  }:${
+    dif.getUTCSeconds() < 10 ? "0" + dif.getUTCSeconds() : dif.getUTCSeconds()
+  }.${
+    dif.getUTCMilliseconds() < 10
+      ? "00" + dif.getUTCMilliseconds()
+      : dif.getUTCMilliseconds()
+  }`;
 }
```

Comment:
avoid this code style, if-else is more readable than multiple conditional operators ? :